internal/cli/ops_manager_agents_list.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package cli
+
+import (
+	"github.com/mongodb/mongocli/internal/flags"
+	"github.com/mongodb/mongocli/internal/json"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+const (
+	AgentType = "AUTOMATION"
+)
+
+type OpsManagerAgentsListOpts struct {
+	*globalOpts
+	store store.AgentLister
+}
+
+func (opts *OpsManagerAgentsListOpts) init() error {
+	var err error
+	opts.store, err = store.New()
+	return err
+}
+
+func (opts *OpsManagerAgentsListOpts) Run() error {
+	var projects interface{}
+	var err error
+	projects, err = opts.store.Agents(opts.projectID, AgentType)
```

Comment:
and by avoid I mean doing
```suggestion
	servers, err := opts.store.Agents(opts.projectID, AgentType)
```